css/style.css

```diff
@@ -4,7 +4,114 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    margin: 0;
+    padding: 0;
+    border: 1px solid white
 }
+*{transition:all 0.3s ease-in-out;}
+*{transition:all 0.3s ease-in-out;}
+
+.container{
+    clear: both;
+    overflow: auto;
+}
+
+nav{float:right;}
+
+.logo img{
+    position: relative;
+    float: left;
+    width:  50px;
+    height: 50px;
+}
+
+ul li{
+  display: inline-block;
+   padding:10px;
+}
+
+ul li:hover{
+  color:orange;
+  
+}
+
+.bckimg{
+    background-image: url(/img/first-background.jpg);
+    height: 500px;
+    width: 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.bckimg p{
+    font-size: 30px;
+    color: white;
+    text-align: center;
+}
+.btn {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
```

Comment:
Nice trick but you shouldn't need this. Change the container (`bckimg`) to use flexbox and center everything inside it to achieve the same result without using `position: absolute;`